pyproject.toml

```
@@ -22,7 +22,20 @@ version = "0.0.1.dev0"
 requires-python = ">=3.8"
 license = {file = "LICENSE"}
 readme = "README.md"
-dependencies = ["anndata","numpy","xarray","zarr","ome_zarr"]
+dependencies = [
+    "anndata",
+    "numpy",
+    "xarray",
+    "zarr",
+    "ome_zarr",
+    "spatial_image",
+    "multiscale_spatial_image",
+    "xarray-schema",
+    "geopandas",
+    "pandera",
+    "pygeos",
+    "shapely @ git+https://github.com/shapely/shapely.git@main",
```

I think this should work

```suggestion
    "geopandas"
    "pandera"
    "shapely==2.0rc1",
```